.ebextensions/enable_cache_headers.conf

```diff
@@ -0,0 +1,6 @@
+<Directory /opt/python/current/app/{{cookiecutter.project_name}}/staticfiles/>
```

Comment:
Does this actually do what we expect or does the project name need to be filled in here?

Reply:
Silly copy and paste will fix